composer.json

```diff
@@ -10,30 +10,29 @@
         }
     ],
     "require": {
-        "php": "^5.3.9|^7.0",
-        "symfony/framework-bundle": "~2.3",
-        "doctrine/phpcr-bundle": "~1.3",
-        "doctrine/phpcr-odm": "~1.3",
-        "symfony-cmf/core-bundle": "1.3.*",
-        "symfony-cmf/content-bundle": "1.3.*",
-        "symfony-cmf/menu-bundle": "2.0.*",
-        "symfony-cmf/block-bundle": "1.3.*",
-        "symfony-cmf/routing-bundle": "1.4.*",
-        "symfony-cmf/media-bundle": "1.3.*"
+        "php": "^5.5.9|^7.0",
+        "symfony/framework-bundle": "^2.7|^3.0",
+        "doctrine/phpcr-bundle": "^1.3",
+        "doctrine/phpcr-odm": "^1.3",
```

Comment:
i think we discussed at some point if we want symfony-cmf/symfony-cmf to still include phpcr. though, we don't have that many orm entities yet so it probably makes sense.